### PR TITLE
Import doc "How To Organize AGM" from homebrew-governance-private

### PR DESCRIPTION
**Pull request by colindean**
The PLC and members workshopped and reviewed this based on past AGMs, especially the 2023 meeting.

- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [X] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] ~~Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).~~
- [ ] ~~Have you successfully run `brew style` with your changes locally?~~
- [ ] ~~Have you successfully run `brew typecheck` with your changes locally?~~
- [ ] ~~Have you successfully run `brew tests` with your changes locally?~~

-----

### CODE REVIEW

**Comment by colindean** [comment]
Markdownlint seems not to know what to do with `- [ ]` per https://github.com/markdownlint/markdownlint/issues/264. I'm not sure what to do other than remove them, which I'd prefer not to do.

File: docs/How-To-Organize-AGM.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+:information_source: _Regenerate the dates for the WHEN lines in the next several headers
+using this quick command:_
+
+    ruby -rdate -e "YEAR=ARGV[0].to_i;puts ([[44,YEAR-1],[49,YEAR-1]]+(1.upto(4).map{|wk|[wk, YEAR]})).map{|wk,yr|Date.commercial(yr,wk).to_s}" 2024
+
+### Three months prior
+
+**When:** Week 44 of YEAR-1 :date: `2023-10-30`
+
+- [ ] LC: Seek venue through previous contacts or RFP.
```
Comment (MikeMcQuaid) [inline docs/How-To-Organize-AGM.md:68]:
```suggestion
* [ ] LC: Seek venue through previous contacts or RFP.
```

does this work/is it valid Markdown? If not: I reckon we should just disable that particular markdownlint rule.

Comment (colindean) [reply docs/How-To-Organize-AGM.md:68]: TIL. I'll change it.

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks again @colindean!

**Pull request merged**